html/statbrowser.css

```
@@ -0,0 +1,227 @@
+body {
+	font-family: Verdana, Geneva, Tahoma, sans-serif;
+	font-size: 12px !important;
+	margin: 0 !important;
+	padding: 0 !important;
+	overflow-x: hidden;
+	overflow-y: scroll;
+}
+
+body.dark {
+	background-color: #131313;
+	color: #b2c4dd;
+	scrollbar-base-color: #1c1c1c;
+	scrollbar-face-color: #3b3b3b;
+	scrollbar-3dlight-color: #252525;
+	scrollbar-highlight-color: #252525;
+	scrollbar-track-color: #1c1c1c;
+	scrollbar-arrow-color: #929292;
+	scrollbar-shadow-color: #3b3b3b;
+}
+
+#menu {
+	background-color: #F0F0F0;
+	position: fixed;
+	width: 100%;
+	z-index: 100;
+}
+
+.dark #menu {
+	background-color: #202020;
+}
+
+#statcontent {
+	padding: 7px 7px 7px 7px;
```

```suggestion
	padding: 7px;
```
This can be adjusted, as we are using 7px for all sides.